tests/shared/UserLogonHandlersTestDefs.js

```diff
@@ -50,67 +50,63 @@ fluid.defaults("gpii.tests.userLogonHandling.testCaseHolder", {
             }
         },
         proximityTriggeredRequest: {
-            type: "gpii.tests.userLogonHandling.proximityTriggered"
+            type: "gpii.tests.userLogonHandlers.proximityTriggered"
         },
         proximityTriggeredRequest2: {
-            type: "gpii.tests.userLogonHandling.proximityTriggered"
+            type: "gpii.tests.userLogonHandlers.proximityTriggered"
         },
         proximityTriggeredRequest3: {
-            type: "gpii.tests.userLogonHandling.proximityTriggered"
+            type: "gpii.tests.userLogonHandlers.proximityTriggered"
         },
         proximityTriggeredRequestOther: {
             type: "kettle.test.request.http",
             options: {
                 path: "/user/sammy/proximityTriggered"
             }
         },
-        logoutRequestOther: {
+        logoutSammyRequest: {
             type: "kettle.test.request.http",
             options: {
                 path: "/user/sammy/logout"
             }
         },
-        loginRequest2: {
+        loginTestUser1Request: {
             type: "kettle.test.request.http",
             options: {
                 path: "/user/testUser1/login"
             }
         },
-        logoutRequest2: {
+        logoutTestUser1Request: {
             type: "kettle.test.request.http",
             options: {
                 path: "/user/testUser1/logout"
             }
+        },
+        logoutNoUserRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/user/noUser/logout"
+            }
         }
     },
     events: {
         timeoutComplete: null
```

Comment:
I see that the name of this event was inherited from the original implementation. It's a good opportunity to rename them all